pkgs/development/compilers/sbcl/default.nix

```diff
@@ -152,6 +154,10 @@ stdenv.mkDerivation (self: {
       # Fail intermittently
       "gc.impure.lisp"
       "threads.pure.lisp"
+    ]
+    ++ lib.optionals stdenv.hostPlatform.isDarwin [
+      # Fails in sandbox
+      "sb-posix.impure.lisp"
```

Comment:
this feels like a separate issue. Do you have a link of a hydra build that is failing? I've been trying to hunt down this issue but I can't find it. Does it only show up in sandboxed builds?